### PR TITLE
Sections 469-504

### PR DESCRIPTION
**Pull request by dbabvwid**
### What?
Sections 469-504

### Anything Else?

[//]: # (for a sample see: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/)

### CODE REVIEW

**Review by Nilstrieb — CHANGES_REQUESTED** [review]
Please edit your PR title to something more descriptive

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+# Hier noch 476 hinzufügen sobald das Kampfsystem existiert
+# Hier noch 477 hinzufügen sobald Resurection und Gameover existieren
+# Hier noch 478 hinzufügen sobald das Shards-System existiert
+# Hier noch 479 hinzufügen sobald das Kampfsystem existiert
+- number: 480
+  content:
+    - text: |
+        Well, tell you what – I’ll teach you a few tricks, and then youcan go away.
+        What do you say?’You’ve got nothing to lose so you accept. Damor was once aTroubadour, and he instructs you in its arts.
+        Gain 1 CHARISMApoint permanently. Afterwards, for a few hours, he lifts the cursethat soured your water so you can make it down the mountainpath safely.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:118]:
```suggestion
        Gain 1 CHARISMA point permanently. Afterwards, for a few hours, he lifts the curse that soured your water so you can make it down the mountainpath safely.
```

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    - if:
+        hasKeyword: Animal
+      then:
+        - turnTo: 369
+      else:
+        - turnTo: 544
+- number: 472
+  content:
+    - text: |
+        Fort Estgard is one of three forts along the wall that runsright along the border.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:38]:
```suggestion
        Fort Estgard is one of three forts along the wall that runs right along the border.
```

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 3]
```diff
@@ -0,0 +1,177 @@
+- number: 469
+  content:
+    - text: |
+        He tells you of his life-long quest to find the ‘Greatest StoryEver Told’.
+        It is said that the god, Sig, in his aspect as the DivineBard, passed this knowledge to a mystic called Damor theHermit.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:5]:
```suggestion
        It is said that the god, Sig, in his aspect as the Divine Bard, passed this knowledge to a mystic called Damor theHermit.
```

**Review by Nilstrieb — CHANGES_REQUESTED** [review]
please fix all the missing spaces

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+        He lives alone in a lost cave, awaiting the day when he can pass on the story to the one who finds him.
+        ‘I am too old now to pursue the quest – so I pass it on to you.
+        I believe he can be found in the foothills of the Spine of Harkun, the mountains to the north.
+        If you find Damor you will become a great Troubadour!’
+    - turnTo: 100
+- number: 470
+  content:
+    - text: |
+        You grab the woman’s wrist and give it a painful twist.
+        You are not being cruel for the sake of it – she was about to steal your money!‘Oh, let me go,’ she pleads, showing you the little infant son she’s carrying in her other arm.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:15]:
```suggestion
        You are not being cruel for the sake of it – she was about to steal your money! ‘Oh, let me go,’ she pleads, showing you the little infant son she’s carrying in her other arm.
```

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        ‘I need the money for my starving children and my poor crippled husband.
+        ’Yellowport is full of wretched like this.
+        With a snarl, you push her away, tossing a coin after her for good measure (cross 1Shard off your Adventure Sheet).
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:18]:
```suggestion
        With a snarl, you push her away, tossing a coin after her for good measure (cross 1 Shard off your Adventure Sheet).
```

File: fabbwled-backend/src/main/resources/sections/469-504.yaml [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+      then:
+        - turnTo: 369
+      else:
+        - turnTo: 544
+- number: 472
+  content:
+    - text: |
+        Fort Estgard is one of three forts along the wall that runs right along the border.
+        Their purpose is to defend Sokara against the ravening man beasts of Nerech, which are constantly trying to break through to raid the interior.
+        The commander of the fort desperately wants to see you.‘My daughter, Alissia, has been kidnapped by the man beasts!’he says.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/469-504.yaml:40]:
```suggestion
        The commander of the fort desperately wants to see you.‘My daughter, Alissia, has been kidnapped by the man beasts!’ he says.
```

**Review by Nilstrieb — CHANGES_REQUESTED** [review]

**Comment by Nilstrieb** [comment]
still many missing spaces

**Review by Nilstrieb — CHANGES_REQUESTED** [review]
there are still missing spaces

**Review by Nilstrieb — APPROVED** [review]
The sun sets over the small town. All words get back into their houses, each having enough space. They rest.

**Review by tamer545 — APPROVED** [review]

**Review by MagischerWolf05 — APPROVED** [review]
sounds good

**Review by lucasmillanm — APPROVED** [review]

**Review by Noeell — CHANGES_REQUESTED** [review]
Please make sure that the checks pass...

**Comment by Nilstrieb** [comment]
@dbabvwid please update your PR

**Review by Nilstrieb — APPROVED** [review]

**Comment by Nilstrieb** [comment]
noel said he approves

**Review by Noeell — APPROVED** [review]
Please link your issues before merging.

**Pull request merged**